aion/web/views/errors.ex

```diff
@@ -0,0 +1,12 @@
+defmodule Aion.ErrorCodesViews do
+  @moduledoc """
+  Module to share common http error codes
+  """
+  use Aion.Web, :controller
+
+  def unauthenticated(conn) do
+    conn
+      |> put_status(401)
```

Comment:
The tab here is [redundant](https://github.com/christopheradams/elixir_style_guide#pipe-operator)